src/components/QuranReader/QuranReaderBody.tsx

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable i18next/no-literal-string */
```

Comment:
nit: does the `useQcfFont(quranReaderStyles.quranFont, verses)` contains literal string?

Reply:
No. It used to thanks for catching this :)